src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/config/studio/administration/config-list.xml

```diff
@@ -21,163 +21,189 @@
 
     For every configuration you'd like to make editable, you need:
         <file>
+            <module />
             <path />
             <title />
             <description />
             <samplePath />
         </file>
 
     The elements are:
-    - path: the path to the file. This path is rooted in /config off the base of the site/blueprint
+    - module: Crafter CMS module
+    - path: the path to the file. This path is rooted in /config/<module> off the base of the site/blueprint
     - title: the title of this file. This should be a key into the string-table in Studio's localization string table
         "base.js", if no entry is found, Studio will use the string you have here as is
     - description: the description of this file.  This should be a key into the string-table in Studio's localization
         string table "base.js", if no entry is found, Studio will use the string you have here as is
     - samplePath: the path to a sample file to help the user update the file
 -->
 <config>
-    <version>5</version>
+    <version>6</version>
     <files>
         <file>
-            <path>/studio/site-config.xml</path>
+            <module>studio</module>
+            <path>site-config.xml</path>
```

Comment:
This is where we should decide on `path` or `location`. If we change `path` to `location` it'll mean more XSLT updates.

Reply:
I think it might be better to make final decision when we switch to yaml

Reply:
I don't want to move forward with two names. Let's change the name to `path`, see my note above.